infrastructure/src/main/java/org/corfudb/infrastructure/management/FailureDetector.java

```diff
@@ -121,7 +126,8 @@ public PollReport poll(
     @VisibleForTesting
     PollReport pollRound(
             long epoch, UUID clusterID, Set<String> allServers, Map<String, IClientRouter> router,
-            SequencerMetrics sequencerMetrics, ImmutableList<String> layoutUnresponsiveNodes) {
+            SequencerMetrics sequencerMetrics, ImmutableList<String> layoutUnresponsiveNodes,
```

Comment:
The signature is getting too big, maybe refactor it into a separate class?

Reply:
There is only one method (`pollReport()`) in FailureDetector class, it's already extracted. I tried to make it smaller, but it didn't work quite well. Maybe next time we would figure out how to make it better